doc/processes/release_process.md

```diff
@@ -116,13 +116,17 @@ determination can be made to either:
 	* Apply a targetted fix to the release branch and retag, or
 	* Rebase the release branch on the master branch if the 
 	changes that have gone in between the initial RC tag and now are safe.
-1. Retag the `RCx` level as `openj9-#releaseNumber#`.  For the `0.8` release this 
+1. Retag the `-rcX` level as `openj9-#releaseNumber#`.  For the `0.8` release this 
 will be `openj9-0.8`.
 1. Create the [github release](https://help.github.com/articles/creating-releases/)
 corresponding to the tagged level.  The release should link to the Eclipse Release 
 document, the release issue, and the AdoptOpenJDK download links.
 1. Open an Eclipse Bugzilla requesting the branch be marked `protected` to prevent 
 commits after the release is complete.
 
-The `java -version` should now show the tagged level.
+Note, use annotated tags (`git tag -a <tag name>`) as they are treated specially by
+git compared to simple tags.
+
+The `java -version` should now show the tagged level and the javacore will also show
```

Comment:
Could mention that the java.vm.version is set to the tag. It shows up in the -version because the -version output shows `(build ${java.vm.version} ...)`